app.js

```diff
@@ -101,7 +101,7 @@ app.get("/twitchAuthResponse", async (req, res) => {
   });
   const responseAuthUser = await authUser.json();
   if (responseAuthUser.message === "Invalid OAuth token")
-    return res.render("sigin");
```

Comment:
This is being addressed in a previous PR change:
Ref: https://github.com/nonbots/nhanify/blob/9f491ee1b193bc33b1c99b5e6b7c494ee0ce9977/app.js#L104

Reply:
I believe this not longer an issue in the current commit.